### DISCUSSION TITLE
QField Version 2.0 - It's possible use a virtual raster that points to a SD Card folder?

### DISCUSSION THREAD

**Discussion by jonnyforestGIS** [opengisch/QField#2698] [Q&A]
Hi to all,
I was working on a project that uses a virtual raster that points to an SD Card folder directory that is a base map of my project in the last Qfield versions. 
It's it possible to use this option in this last version? Or is a related issue with this blog entry https://www.opengis.ch/2022/03/05/qfield-users-sit-down-we-need-to-talk-about-storage-access-on-android%ef%bf%bc/

I'm using the sd card to store big coverage maps because sometimes I work in places (closed forests, deep valleys) without networks and I need to cover large areas with dispersing plots. To avoid large portable projects, I store the base map raster layers (ex.: aerial image) in a folder of the sd card and create a virtual raster layer.

This option can solve my problem?
https://docs.qfield.org/how-to/outside-layers/

If you have some advice or an easy way to do it in a different way can you share it?

Thank you for your time.

Cheers

**Comment by suricactus** [comment]
Short answer is no, as described in the blog post.

The longer answer is you can use localized data paths, which is what you see with the "Outside layers" option and this way you can share one layer across multiple projects.

**Reply by jonnyforestGIS** [reply]
Thank you @suricactus.
Cheers
